clients/cobol-lsp-vscode-extension/src/services/LanguageClientService.ts

```diff
@@ -43,11 +43,19 @@ export class LanguageClientService {
     }
 
     public start(): vscode.Disposable {
-        const languageClient = new LanguageClient(LANGUAGE_ID,
+        const languageClient = this.getLanguageClient();
+        return languageClient.start();
+    }
+
+    public stop(): Thenable<void> {
+        return this.getLanguageClient().stop();
+    }
+
+    private getLanguageClient() {
+        return new LanguageClient(LANGUAGE_ID,
```

Comment:
Doesn't it mean that it will stop a new `LanguageClient` instance?

Reply:
yes, this is the wrong implementation.

Reply:
updated